cmd/ocm-backplane/login/login.go

```diff
@@ -73,6 +74,14 @@ func init() {
 		"Save kube configuration in the specific path when login to multi clusters.",
 	)
 
+	flags.StringVarP(
+		&args.defaultNamespace,
+		"namespace",
+		"n",
+		"openshift-backplane-srep",
```

Comment:
I don't think it's good to set the default NS as `open shift-backplane-step` always because the default NS changes based on the user role.
```yaml
  - id: "SREP"
    saNamespace: "openshift-backplane-srep"
  - id: "CEE"
    saNamespace: "openshift-backplane-cee"
```
If `CEE` login to the cluster `openshift-backplane-srep` NS  may not exist

Reply:
> I don't think it's good to set the default NS as `open shift-backplane-step` always because the default NS changes based on the user role.

I agree with your thought.

> 
> ```yaml
>   - id: "SREP"
>     saNamespace: "openshift-backplane-srep"
>   - id: "CEE"
>     saNamespace: "openshift-backplane-cee"
> ```

I did not spot this mapping inside the backplane-cli repo .  
How can my program fetch this mapping dynamically and set the default namespace accordingly ?

How can we make the program future-ready for scenarios like addition of new user roles mapped to different namespace or,  organizations preventing these user roles  from setting their context to these default namespace at the time of login ?

Reply:
Hi @amej 
pls go through the  [slack](https://redhat-internal.slack.com/archives/C016S65RNG5/p1709798020550339) thread.